enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/MobileTokenService.java

```diff
@@ -58,13 +56,15 @@
 public class MobileTokenService {
 
     private static final int OPERATION_LIST_LIMIT = 100;
+
     private static final String ATTR_ACTIVATION_ID = "activationId";
     private static final String ATTR_APPLICATION_ID = "applicationId";
     private static final String ATTR_IP_ADDRESS = "ipAddress";
     private static final String ATTR_USER_AGENT = "userAgent";
     private static final String ATTR_AUTH_FACTOR = "authFactor";
     private static final String ATTR_REJECT_REASON = "rejectReason";
-    private static final String PROXIMITY_OTP = "proximity_otp";
+    private static final String ATTR_PROXIMITY_OTP = "proximity_otp";
+    private static final String ATTR_MOBILE_TOKEN_DATA = "mobileTokenData";
```

Comment:
ℹ️ Under this key, the enrollment additional data is stored in PAS `operation#additionalData`, so the final path will be `operation.additionalData.mobileTokenData.*`